cmd/crictl/status.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli"
+	"golang.org/x/net/context"
+	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+)
+
+var runtimeStatusCommand = cli.Command{
+	Name:  "status",
+	Usage: "get the status of runtime",
+	Action: func(context *cli.Context) error {
+		// Set up a connection to the server.
```

Comment:
@feiskyer @heartlock 

We have `After` see [urfave/cli/#151](https://github.com/urfave/cli/pull/151) . And we also have companion action of `After` which called `Before`.